StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -330,13 +330,7 @@ extension PaymentSheet {
                 }
                 intentPromise.resolve(with: .paymentIntent(paymentIntent))
             }
-            let additionalParameters = [
-                "merchant_support_async": configuration.allowsDelayedPaymentMethods,
-                "merchant_support_shipping": configuration.allowsPaymentMethodsRequiringShippingAddress,
-            ]
-            configuration.apiClient.retrievePaymentIntentWithPreferences(
-                withClientSecret: clientSecret,
-                additionalParameters: additionalParameters) { result in
+            configuration.apiClient.retrievePaymentIntentWithPreferences(withClientSecret: clientSecret) { result in
```

Comment:
in `retrievePaymentIntentWithPreferences`, we added `additionalParameters` as a parameter, can we get rid of that as well?